modules/rubiconBidAdapter.js

```diff
@@ -638,7 +638,7 @@ export const spec = {
               bidObject.adserverTargeting = extPrebidTargeting;
             }
 
-            // try to get cache values from 'response.ext.prebid.cache'
+            // try to get cache values from 'response.ext.prebid.cache.js'
```

Comment:
is this change necessary?

Reply:
nope, that was an accidental catch by the regex replace